robot/Cumulus/resources/NPSP.py

```diff
@@ -1279,4 +1286,50 @@ def verify_toast_message(self,value):
                 found=True
                 print("Toast message verified")
                 break
-        assert found, "Expected Toast message not found on page"
+        assert found, "Expected Toast message {} not found on page".format(value)
+
+    def edit_record_field_value(self,field,value):
```

Comment:
This is a good example of why I like robot - sometimes the simplest user action requires lots of minor details, like scrolling an element into view. This is a cool keyword because it hides a bunch of complexity behind an easy-to-understand keyword.